test/FsAutoComplete.IntegrationTests/DotNetCore/sample1/c1/c1.fsproj

```diff
@@ -0,0 +1,15 @@
+<Project Sdk="FSharp.NET.Sdk;Microsoft.NET.Sdk">
+  <PropertyGroup>
+    <OutputType>Exe</OutputType>
+    <TargetFramework>netcoreapp1.0</TargetFramework>
+  </PropertyGroup>
+  <ItemGroup>
+    <Compile Include="Program.fs" />
+  </ItemGroup>
+  <ItemGroup>
+    <PackageReference Include="FSharp.NET.Sdk" Version="1.0.*" PrivateAssets="All" />
+    <PackageReference Include="FSharp.Core" Version="4.1.*" />
+    <ProjectReference Include="..\l1\l1.fsproj" />
+  </ItemGroup>
+  <Import Project="..\..\..\..\..\.paket\Paket.Restore.targets" />
```

Comment:
can you remove `Paket` target for now? so we can test a normal `dotnet new` project.
we can add paket later. just to remove a possibile failure